index.html

```diff
@@ -10,7 +10,57 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <main>
+        <header>
+            <h1>Weather Report</h1>
+        </header>
+    
+        <h3>For the lovely city of Seattle</h3>
+        <div id="temp-city">
+        <section class="temp_section">
+                <section id="temp_display">Temperature</section>
+                    <div class="temp-change-buttons_div"> 
+                        <button id="up-arrow" type="button">⬆️ </button>
+                        <div id="temp-count">0</div>
+                        <button id="down-arrow" type="button">⬇️</button>
+                    </div>
+                </section>
+                <section class="city-query">
+                    <label for="city-name">🏙️ City Name 🏙️</label>
+                    <input type="text" id="city-name" size="20">
+                    <span>
+                        <button id="realtime-button-weather" type="button" alt="Get Realtime Temperature">Get Realtime Temperature</button>
+                        <button id="reset-city-name" alt="Reset Temperature">Reset</button></span>
+                    </input>
+                </section>
+        </section>
+        </div>
+
+        <section class="garden-section">
+        <h2>Weather Garden</h2>
+            <div id="landscape_display" class="garden_content">
+                <div id="sky">🌞</div>
+                <div id="landscape">🌵🌼🌸🍁🌵🌼🌸🍁</div>
```

Comment:
Best practice is to let the JS handle populating the initial displays here instead of setting them directly in the HTML!